examples/ding-dong-bot.py

```diff
@@ -1,21 +1,20 @@
 """doc"""
-import os
 import asyncio
 import logging
+import os
 from typing import Optional, Union
 
-from wechaty_puppet import PuppetOptions, FileBox  # type: ignore
-from wechaty_puppet_hostie import HostiePuppet  # type: ignore
+from wechaty_puppet import FileBox  # type: ignore
 
 from wechaty import Wechaty, Contact
 from wechaty.user import Message, Room
```

Comment:
It would be better if we can import all class from just `wechaty` instead of from other deep import path like wechaty.user etc.

Reply:
I'll make this change at next pr.